main.go

```diff
@@ -35,9 +36,15 @@ var (
 	flagDebug       = flag.Bool("debug", false, "Enables debug messages")
 	flagInsecure    = flag.Bool("insecure", false, "Uses unencrypted connections when fetching images")
 	flagCompression = flag.String("compression", "gzip", "Type of compression to use; allowed values: gzip, none")
+	flagVersion     = flag.Bool("version", false, "Print version")
 )
 
-func runDocker2ACI(arg string, flagNoSquash bool, flagImage string, flagDebug bool, flagInsecure bool, flagCompression string) error {
+func printVersion() {
+	fmt.Println("docker2aci version", docker2aci.Version)
+	fmt.Println("appc version", docker2aci.AppcVersion)
+}
+
+func runDocker2ACI(arg string, flagNoSquash bool, flagImage string, flagDebug bool, flagInsecure bool, flagCompression string, flagVersion bool) error {
```

Comment:
Besides, passing a `flagVersion` parameter here is not necessary, you don't use it here at all.